components/operator/internal/modules/service.go

```diff
@@ -326,6 +326,7 @@ type Service struct {
 	InjectPostgresVariables bool
 	HasVersionEndpoint      bool
 	Liveness                Liveness
+	LivenessEndpoint        *string
```

Comment:
Pointer is useless there, an empty string is enough to define than the endpoint is not overriden